chaincode-labs/chaincode-residency/2019-09-18-christian-decker-multiparty-channels.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Multiparty Channels (Lightning Network) with Christian Decker (2019-09-18)
```

Comment:
This date is actually 2019-06-28 but impossible to know that from the video alone. Would you mind changing the file name to match?